client.go

```diff
@@ -107,7 +109,28 @@ func (cs *clientSession) run() (err error) {
 	cs.dc.OnOpen(cs.dataChannelOnOpen())
 	cs.dc.OnMessage(cs.dataChannelOnMessage())
 
-	if cs.offer, err = sd.Decode(cs.offerString); err != nil {
+	offerText := cs.offerString
+	if len(cs.offerString) == 32 {
+		url := fmt.Sprintf("https://www.10kb.site/%s", cs.offerString)
+		req, err := http.NewRequest("GET", url, nil)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
have this function return an error, we can handle exiting elsewhere

Reply:
also likely good to annotate this error: `errors.Wrap(err, "error requesting www.10kb.site")`